src/main/java/com/flightstats/hub/model/LargeContent.java

```diff
@@ -3,17 +3,16 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-import com.flightstats.hub.app.HubProvider;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.flightstats.hub.app.HubBindings;
+import lombok.extern.slf4j.Slf4j;
 
 import java.io.IOException;
 
+@Slf4j
 public class LargeContent {
 
     static final String CONTENT_TYPE = "application/hub";
-    private static final Logger logger = LoggerFactory.getLogger(LargeContent.class);
-    private static final ObjectMapper mapper = HubProvider.getInstance(ObjectMapper.class);
+    private static final ObjectMapper mapper = HubBindings.objectMapper();
```

Comment:
This should be injected to maintain consistency across the app.

Reply:
Yeah, my preference was to switch this to an injected singleton, but guicification was starting to snowball.  I can go do that though.
Do we have a general route we're going here?  I see lots of cases where the code pulls an instance manually from the Injector through HubProvider.  Not crazy about that pattern.

Reply:
We're trying to deprecate HubProvider, so your instinct not to use it is appreciated. Lemme take a look at the code and think a little before I answer the question about whether we have a general route. In the meantime, maybe someone who's feeling more coherent has an answer more readily available.

Honestly, if it was starting to snowball, I'd be cool with splitting that refactoring out to a different PR. Makes it easier to revert (especially if we squash-merge our PRs).

Reply:
It's not that bad.  I agree it's definitely better this way.  I'll toss it in.  CCS is the only consumer, so it doesn't reach that far.  Will push again to DI tonight though since there's been a bit of churn.

Reply:
Eww, all the methods in here are static. I'd probably make them instance methods and start injecting the ObjectMapper here and a LargeContent dependency in `ClusterContentService`...which I think is the only place it's used. Does that seem like it'd work or is that oversimplified?

Reply:
Plus one to doing this in another PR, seems a little out of scope

Reply:
Ah...sorry...just sent it up.